tests/integration/release/test_release.py

```diff
@@ -14,131 +15,114 @@
 
-    run_column_noising_tests(
-        dataset_name, config, fuzzy_checker, check_noised, check_original, shared_idx
-    )
+    # update parameters
+    if source is None:
+        source = paths.SAMPLE_DATA_ROOT
+    elif isinstance(source, str) or isinstance(source, Path):
+        source = Path(source)
+        validate_source_compatibility(source, dataset_schema)
 
+    engine = get_engine_from_string(engine_name)
 
-@pytest.mark.parametrize("expected_noise", ["default", 0.01])
-def test_omit_row(
-    expected_noise: str | float,
-    unnoised_dataset: Dataset,
-    dataset_name: str,
-    fuzzy_checker: FuzzyChecker,
-) -> None:
-    """Tests that omit_row noising is being applied at the expected proportion."""
-    original_data = unnoised_dataset.data
-    config_dict = get_single_noise_type_config(dataset_name, NOISE_TYPES.omit_row.name)
+    data_file_paths = get_dataset_filepaths(Path(source), dataset_schema.name)
+    filters = get_generate_data_filters(dataset_schema, year, state)
+    unnoised_data = [load_standard_dataset(path, filters, engine) for path in data_file_paths]
 
-    if expected_noise != "default":
-        config_dict[dataset_name][Keys.ROW_NOISE][NOISE_TYPES.omit_row.name][
-            Keys.ROW_PROBABILITY
-        ] = expected_noise
-        config = NoiseConfiguration(LayeredConfigTree(config_dict))
+    if engine == DASK_ENGINE:
+        # TODO: [MIC-5960] move this compute to later in the code
+        dataset_data: list[pd.DataFrame] = [data.compute() for data in unnoised_data if len(data) != 0]  # type: ignore [operator]
     else:
-        config = NoiseConfiguration(LayeredConfigTree(config_dict))
-        # updating expected_noise from 'default' to actual default value
-        expected_noise = config.get_row_probability(dataset_name, NOISE_TYPES.omit_row.name)
-    assert isinstance(expected_noise, float)
+        dataset_data = [data for data in unnoised_data if len(data) != 0]  # type: ignore [misc]
 
-    dataset_schema = DATASET_SCHEMAS.get_dataset_schema(dataset_name)
-    dataset = Dataset(dataset_schema, original_data, SEED)
-    NOISE_TYPES.omit_row(dataset, config)
-    noised_data = dataset.data
+    has_small_shards = dataset_name == "acs" or dataset_name == "cps" or dataset_name == "wic"
+    if str(source) == RI_FILEPATH and has_small_shards:
+        dataset_data = [pd.concat(dataset_data).reset_index()]
 
-    fuzzy_checker.fuzzy_assert_proportion(
-        name="test_omit_row",
-        observed_numerator=len(original_data) - len(noised_data),
-        observed_denominator=len(original_data),
-        target_proportion=expected_noise,
-    )
+    datasets = [Dataset(dataset_schema, data, SEED) for data in dataset_data]
 
-    assert set(noised_data.columns) == set(original_data.columns)
-    assert (noised_data.dtypes == original_data.dtypes).all()
+    for noise_type in NOISE_TYPES:
+        prenoised_dataframes = [dataset.data.copy() for dataset in datasets]
```

Comment:
```suggestion
        pre_noise_dataframes = [dataset.data.copy() for dataset in datasets]
```

Perhaps?

Reply:
Interesting...my gut reaction was that I definitely want there to be a "d" at the end. But the more I sit with it the less I care.